packages/core/src/components/steps-modal/stories/steps-modal.args.ts

```diff
@@ -146,4 +146,6 @@ export const ModalComponentArgs = {
   currentStep: 1,
   isOpen: false,
   closeOnFinish: false,
+  primaryText: 'Primary',
+  secondaryText: 'Secondary',
```

Comment:
we could be more descriptive here

```suggestion
  primaryText: 'Next',
  secondaryText: 'Cancel',
```